scm/driver/gitlab/content.go

```diff
@@ -19,23 +19,12 @@ type contentService struct {
 }
 
 func (s *contentService) Find(ctx context.Context, repo, path, ref string) (*scm.Content, *scm.Response, error) {
-	path = url.QueryEscape(path)
-	path = strings.Replace(path, ".", "%2E", -1)
-	endpoint := fmt.Sprintf("api/v4/projects/%s/repository/files/%s?ref=%s", encode(repo), path, ref)
-	out := new(content)
+	endpoint := fmt.Sprintf("api/v4/projects/%s/repository/files/%s/raw?ref=%s", encode(repo), url.QueryEscape(path), ref)
+	out := new(bytes.Buffer)
 	res, err := s.client.do(ctx, "GET", endpoint, nil, out)
-	raw, berr := base64.StdEncoding.DecodeString(out.Content)
```

Comment:
Why has this been dropped?

Reply:
Because I replaced the original API call (https://docs.gitlab.com/ee/api/repository_files.html#get-file-from-repository) with another call that can directly return the raw data (see Line 22): https://docs.gitlab.com/ee/api/repository_files.html#get-raw-file-from-repository